liberty-ls/src/main/java/io/openliberty/tools/langserver/model/propertiesfile/PropertiesKeyInstance.java

```diff
@@ -66,6 +66,11 @@ public CompletableFuture<List<CompletionItem>> getCompletions(Position position)
     public List<CompletionItem> getValidValues() {
         List<String> values = ServerPropertyValues.getValidValues(textDocumentItem, propertyKey);
         List<CompletionItem> results = values.stream().map(s -> new CompletionItem(s)).collect(Collectors.toList());
+        // Preselect the default.
+        // This uses the first item in the List as default. 
+        // (Check ServerPropertyValues.java) Currently sorted to have confirmed/sensible values as default.
+        CompletionItem first = results.get(0);
+        first.setPreselect(true);
```

Comment:
Does this mean the first item is selected even if you said there was no default (as in the case of WLP_DEBUG_REMOTE)?

Reply:
Yes, it will preselect the first item. This is where it gets a bit spicy. 

While technically there's no default for `WLP_DEBUG_REMOTE`, by not defining it, remote debugging is not allowed (essentially a `n` default). 
But reasonably, I think anyone adding this property would be using it to enable remote debugging, so they would want to define it with `y` which is the current preselected value.

Would it make more sense to preselect the default behavior? or the intended/desired behavior by someone adding this property?

Reply:
So you confirmed what I thought would happen, and it is a bit tricky which value to preselect. But I think it is reasonable to select `y`. Let's make it clear in the comment though that `y` will get preselected for this property.